docs/source/terraform/index.rst

```diff
@@ -0,0 +1,133 @@
+
+    api_gateway_lambda_role_arn         = "<AWS_IAM_Role_ARN>"
+    lambda_ingress_service_iam_role_arn = "<AWS_IAM_Role_ARN>"
+    lambda_authorizer_iam_role_arn      = "<AWS_IAM_Role_ARN>"
+
+    nucleus_dum_cognito_initial_users = [
+      {
+        "username": "<Username>",
+        "password": "<Password>",
+        "group": "<PDS_Group_Name>"
+        "email": "<Email address>"
+      },
+      ...
+    ]
+
+Each of the bracketed values above must be filled in with an appropriate value
+prior to deployment. Descriptions of each field follow:
+
+* ``<AWS_Profile_Name>``: The AWS Profile/Account name of the environment to deploy to.
+* ``<Path_to_local_aws_credentials_file>``: Path to a local file (such as ``~/.aws/credentials``) containing a valid set of AWS credentials_ for the specifed profile.
+* ``<S3_Bucket_Name>``: Name of an S3 bucket which Terraform will use to stage intermediate build products. The bucket will be created by Terraform and should not already exist.
+* ``<VPC_ID>``: ID of the Virtual Private Cloud instance where the deployed service will reside.
+* ``<AWS_IAM_Role_ARN>``: Amazon Resource Name (ARN) for the AWS Identity Access Management Role which grants the required permissions for the DUM server components to communicate.
+
+.. note::
+  * The selected Role(s) must include both ``lambda.amazonaws.com`` and ``apigateway.amazonaws.com`` as "Trusted Entities".
+  * The policy attached to the Role(s) must also grant adequate permissions for the following services: API Gateway, Cloudwatch, Cloudwatch Logs, Lambda, and S3
+
+* The `nucleus_dum_cognito_initial_users` can be populated with a list of dictionaries defining a set of user accounts to
+  populate the Cognito user pool with upon creation. An arbitrary number of initial users can be provided.
+  * The ``<Username>`` and ``<Password>`` fields can be anything, but the password must conform to the Password policy set on the Cognito user pool.
+  * For ``<PDS_Group_Name>``, one of the following may be used: ``PDS_ATM_USERS, PDS_ENG_USERS, PDS_GEO_NODE, PDS_IMG_NODE, PDS_NAIF_NODE, PDS_PPI_NODE, PDS_RS_NODE, PDS_RMS_NODE, PDS_SBN_NODE``
+  * ``<Email address>`` should be a valid email address for the user, as this is where things such as password resets will be sent.
+
+Deploying the Terraform
+-----------------------
+
+Once the user `.tfvars` is created and saved locally, run the following commands to deploy the DUM service with terraform::
+
+    terraform init
+    terraform plan -var-file="<path to user .tfvars file>" -out="pds-dum.tfplan"
+    terraform apply "pds-dum.tfplan"
+
+If everything is configured correctly, deployment should only take about 5-10 minutes.
+
+Utilizing Terraform Outputs
+---------------------------
+
+On successful deployment, Terraform will output a number of key/value pairs with
+information needed to configure the INI config of the client-side ``pds-ingress-client``
+application.
+
+A sample of this output follows:
```

Comment:
Double-colon `::` at the end of this line? It'll make the following lines appear in fixed-width font.